core/templates/dev/head/services/ResourceService.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service for Resources.
+ */
+
+oppia.factory('ResourceService', ['$http',
+  function($http) {
+    return {
+      getPromoBar: function() {
```

Comment:
You're not really getting the promo bar object itself here -- so maybe rename to getPromoBarData, getPromoBarDetails, or similar.

Reply:
Done.